packages/@angular/cli/commands/serve.ts

```diff
@@ -23,6 +23,7 @@ export interface ServeTaskOptions extends BuildOptions {
   sslCert?: string;
   open?: boolean;
   hmr?: boolean;
+  app?: string;
```

Comment:
Remove app from here and instead add it to https://github.com/angular/angular-cli/blob/master/packages/@angular/cli/models/build-options.ts.